tests/integration/SdkTest_test.cpp

```diff
@@ -5035,10 +5053,10 @@ TEST_F(SdkTest, SyncResumptionAfterFetchNodes)
 
     LOG_verbose << " SyncResumptionAfterFetchNodes : syncying folders";
 
-    int tag1 = syncFolder(sync1Path);
+    int tag1 = syncFolder(sync1Path);  tag1;
```

Comment:
Out of curiosity, what is this for?

Reply:
to avoid unused var warnings I guess :-D

Reply:
IIRC that won't work when optimizations are enabled, i.e. Release builds will still get the warnings.
Using `(void)tag1;` should work for all cases.

Reply:
wouldn't it? Interesting ... I would have though those warnings were reported before the optimizations take place :thinking:

Reply:
Probably (only?) the NOPs are optimized earlier.
At the end of the optimized build unused variables won't be present at all in the final executable, but the warning is there even for those cases, so not all optimization have been applied at that point.